src/main/react/src/assets/icons/index.js

```diff
@@ -0,0 +1,273 @@
+import { sygnet } from './sygnet'
+import { logo } from './logo'
+import { logoNegative } from './logo-negative'
+
+import {
+  cibSkype,
+  cibFacebook,
+  cibTwitter,
+  cibLinkedin,
+  cibFlickr,
+  cibTumblr,
```

Comment:
Does this affect page loading time much? We should probably remove things we'd never use here.